source/codegen/metadata/niscope/attributes.py

```diff
@@ -1,2556 +1,1558 @@
-        },
-        'lv_property': 'Waveform Measurement:Reference Levels:Channel Based Low Ref Level',
+        'grpc_type': 'double',
         'name': 'MEAS_CHAN_LOW_REF_LEVEL',
         'resettable': True,
         'type': 'ViReal64'
     },
     1150039: {
-        'access': 'read-write',
-        'channel_based': True,
         'codegen_method': 'public',
-        'documentation': {
-            'description': '\nStores the mid reference level used in many scalar measurements. Different channels may have different reference levels. Do not use the IVI-defined, nonchannel-based attributes such as NISCOPE_ATTR_MEAS_MID_REF if you use this attribute to set various channels to different values.\nDefault: 50%\n'
-        },
-        'lv_property': 'Waveform Measurement:Reference Levels:Channel Based Mid Ref Level',
+        'grpc_type': 'double',
         'name': 'MEAS_CHAN_MID_REF_LEVEL',
         'resettable': True,
         'type': 'ViReal64'
     },
     1150040: {
-        'access': 'read-write',
-        'channel_based': True,
         'codegen_method': 'public',
-        'documentation': {
-            'description': '\nStores the high reference level used in many scalar measurements. Different channels may have different reference levels. Do not use the IVI-defined, nonchannel-based attributes such as NISCOPE_ATTR_MEAS_HIGH_REF if you use this attribute to set various channels to different values.\nDefault: 90%\n'
-        },
-        'lv_property': 'Waveform Measurement:Reference Levels:Channel Based High Ref Level',
+        'grpc_type': 'double',
         'name': 'MEAS_CHAN_HIGH_REF_LEVEL',
         'resettable': True,
         'type': 'ViReal64'
     },
     1150041: {
-        'access': 'read-write',
-        'channel_based': True,
         'codegen_method': 'public',
-        'documentation': {
-            'description': '\nSpecifies the width of bandpass and bandstop type filters in hertz. The cutoff frequencies occur at NISCOPE_ATTR_MEAS_FILTER_CENTER_FREQ ± one-half width.\nDefault: 1.0e3 Hz\n'
-        },
-        'lv_property': 'Waveform Measurement:Filter:Width',
+        'grpc_type': 'double',
         'name': 'MEAS_FILTER_WIDTH',
         'resettable': True,
         'type': 'ViReal64'
     },
     1150042: {
-        'access': 'read-write',
-        'channel_based': True,
-        'codegen_method': 'public',
-        'documentation': {
-            'description': '\nSpecifies the FIR window type. The possible choices are:\nNISCOPE_VAL_NONE\nNISCOPE_VAL_HANNING_WINDOW\nNISCOPE_VAL_HAMMING_WINDOW\nNISCOPE_VAL_TRIANGLE_WINDOW\nNISCOPE_VAL_FLAT_TOP_WINDOW\nNISCOPE_VAL_BLACKMAN_WINDOW\nThe symmetric windows are applied to the FIR filter coefficients to limit passband ripple in FIR filters.\nDefault: NISCOPE_VAL_NONE\n'
-        },
-        'enum': 'FIRFilterWindow',
```

Comment:
Missing 'enum' tag for this attribute. You can look [here](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/358221?_a=files&path=/src/nidmm/nidmmBaseDistribution/nidmmAPIMetadata/source/nidmmAPIMetadata/grpc_device/attributes/attributes.hapigen) for an example of how to add these tags.